mouse/record.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+
+
+import argparse
+import io
+import time
+import picamera
+import datetime
+import os
+
+
+
+def main(*argv):
+   """
+       Defines arguments: time before and after, and mouse/trial to be tested
+
+   """
+   
+   
+   parser = argparse.ArgumentParser(description='seconds before and after lever is pressed.')
+   parser.add_argument('before', type=int, help='seconds to record before.')
+   parser.add_argument('after', type=int, help='seconds to record after.')
+   parser.add_argument('folder', type=str, default=".", help = 'mouse to be tested')
+   args = parser.parse_args(argv[1:])
+   folder = args.folder
+   x = args.before
+   y = args.after
+   z = x + y
+   import RPi.GPIO as GPIO
+   GPIO.setmode(GPIO.BCM)
+   GPIO.setup(27, GPIO.IN, GPIO.PUD_UP)
+
+   with picamera.PiCamera(framerate=90) as camera:
+      try:
+         if not os.path.exists(folder):
+            os.makedirs(folder)
+         stream = picamera.PiCameraCircularIO(camera, seconds = z)
+         while True:
+            stream.seek(0)
+            camera.start_recording(stream, format= 'h264', splitter_port=1)
+            GPIO.wait_for_edge(27, GPIO.FALLING)
+            camera.wait_recording(y, splitter_port=1)
+            camera.stop_recording()
+            '''stream.seek(z*90)'''
```

Comment:
Is this line still needed? It seems like the answer from our discussion earlier is no. If so, delete it in one commit and explain in the message why it can go away. Note this line as well ( https://github.com/DudLab/mouse_record/pull/1/files#diff-e232b0d1acf7cc0c10b99de2ef8f1377R40 ).